src/app/components/MediaLoader/index.tsx

```diff
@@ -167,7 +170,11 @@ const MediaContainer = ({
           ? styles.audioMediaContainer
           : styles.standardMediaContainer
       }
-    />
+    >
+      <noscript>
+        <Message message={noJsMessage} />
+      </noscript>
```

Comment:
Should this always have _some_ kind of fallback (e.g. an english version of the message)? I thought it would be the `translations?.media?.noJs` but the optional chaining implies we might not always have a translation. 
If not, maybe this should conditionally render based on whether there is a message passed in at all?
```suggestion
      ...(noJsMessage && {
            <noscript>
              <Message message={noJsMessage} />
            </noscript>
           }
         )
```

Reply:
Let me have a think, and look at what we do elsewhere in the application. I think an English fallback would be better than not seeing the component at all, but let me check.

Reply:
So I did a bit of digging around on this.

The fallback translation is coming from this line: https://github.com/bbc/simorgh/blob/14ca6bd8fbf1ba12de64f86795998008679f90d4/src/app/components/MediaLoader/index.tsx#L250-L251, and is needed in case there was no noJS message set in the placeholder config (which can sometimes happen if the media type doesn't have a placeholder config e.g. live radio). 

You can also see that it uses optional chaining, so I quickly updated the type definition such that `translations.media.noJs` was mandatory, and I got a TS failure because the `scotland` and `archive` services did not have this defined 🫨. Annoyingly neither of these services are officially supported anymore (see https://jira.dev.bbc.co.uk/browse/WORLDSERVICE-784), but for the purposes of this PR (& scope creep avoidance), I am going to leave it be. 

So, we will always have a fallback message - even for news/sport, so we don't need to worry about conditionally rendering the component based on the presence of `noJsMessage`